go/vt/vtctl/schematools/reload.go

```diff
@@ -67,8 +68,15 @@ func ReloadShard(ctx context.Context, ts *topo.Server, tmc tmclient.TabletManage
 			defer wg.Done()
 
 			if concurrency != nil {
-				concurrency.Acquire()
-				defer concurrency.Release()
+				if concurrency.Acquire(ctx, 1) != nil {
+					// We timed out waiting for the semaphore. This is best-effort, so just log it and move on.
```

Comment:
Should we print out the error here?

Reply:
We can, I can add it. It would be a context cancelled or expired message always but that might be helpful still?

Reply:
If the error is always the same, then I think the warning log is sufficient as the error itself is not going to bring much more help.